exchanges.yml

```
@@ -358,6 +358,16 @@ components:
           description: An optional step directive that tells the exchange to handle challenge management via a VC API verifier service it has a zcap for.
         verifiablePresentationRequest:
           $ref: "./components/VerifiablePresentationRequest.yml#/components/schemas/VerifiablePresentationRequest"
+        presentationSchema:
+          type: object
+          description: The presentation schema to use when validating a presentation in the current step.
+          properties:
+            types:
+              type: string
+              description: The type of presentation schema, such as <code>JsonSchema</code>.
+            jsonSchema:
+              type: object
```

```suggestion
            schema:
              type: string
```

This was discussed on the 2025-06-24 telecon as an alternative.

---

Fixed in 677ec30771faccbe662e2ea9e0df13c38bb37fd7.